sympy/parsing/latex/__init__.py

```diff
@@ -0,0 +1,25 @@
+from sympy.external import import_module
+
+from .errors import LaTeXSyntaxError  # noqa
+
+
+def parse_latex(s):
+    """Converts the string ``s`` to a SymPy ``Expr``
```

Comment:
Add a note here that the input string should be prepended with an `r` so that the latex backslashes are interpreted correctly.

Reply:
Also the disclaimers I discussed should be in this docstring, as well as the main docs.